frontend/scripts/metrica.js

```diff
@@ -9,12 +9,40 @@
             });
         }
     }
+
+    window.onerror = (errorMsg, errorUrl) => {
+        const url = state.host_api + state.routes.create_error;
+        const tracking_number = window._metricaTracking.getObjectMetricaConf()[1][1];
+        const token = window._metricaTracking.getToken();
+        let headers = {
+            'Content-Type': 'application/json',
+            'x-website': tracking_number,
+        };
+        let data = {
+            message: errorMsg,
+            stack_trace: errorUrl,
+            page: window._metricaTracking.getPage(),
+            page_title: window._metricaTracking.getTitle(),
+
+        };
+
+        if (token) {
+            headers = {
```

Comment:
You can do it like this

{
                ...headers,
                'x-visitor': 'Bearer ' + token
            }